enhancements/authentication/unauthenticated-subjects.md

```diff
@@ -0,0 +1,137 @@
+---
+title: hardening-openshift-apiserver-default-policy-resource
+authors:
+  - "@vareti"
+reviewers:
+  - "@marun"
+  - "@sttts"
+  - "@deads2k"
+approvers:
+  - "@sttts"
+creation-date: 2020-05-07
+last-updated: 2020-05-13
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Hardening OpenShift-apiserver Default Policy Resources on Upgrade Clusters
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+* In earlier releases of OpenShift, the authorization module would allow anonymous users to access discovery endpoint.
+* This enhancement proposes a way to remove unauthenticated subjects from cluster role bindings after an upgrade from this earlier release.
+* It is possible that a customer is relying on anonymous user behavior in an implementation.
+* This enhancement also proposes a way to opt-out of this reconciliation mechanism. Thus helping cases where changes are not desirable to customer implementations.
+* Later releases of OpenShift removed this access for anonymous users. So, this enhancement would not affect the customers who install a fresh cluster using these later releases.
```

Comment:
can you link where we changes that? Was either me or Mo, probably in origin repository.